up42/auth.py

```diff
@@ -65,11 +71,23 @@ def is_http_429_error(exception):
 
 
 class Auth:
+    cred_mapping = {
+        AuthType.PROJECT.value: {
+            "credentials_id": "project_id",
+            "credentials_key": "project_api_key",
+        },
+        AuthType.ACCOUNT.value: {
+            "credentials_id": "username",
+            "credentials_key": "password",
+        },
+    }
+
     def __init__(
         self,
+        auth_type: str = AuthType.PROJECT.value,
         cfg_file: Union[str, Path, None] = None,
-        project_id: Optional[str] = None,
-        project_api_key: Optional[str] = None,
+        credentials_id: Optional[str] = None,
```

Comment:
ahh, so users will have use these keywords now.
Hmm, this will force them to make changes to their pipeline.
Since we will be deprecating it in future anyway, I would suggest to keep the existing keys.

Reply:
Yes, I added support for the previous ones. :)